hms-app/src/components/input-forms/HackathonForm.tsx

```diff
@@ -18,11 +18,13 @@ type IProps = {
   context: string
   hackathonId: string | null
   closeAccordion?: () => void
+  reload?: () => void
+  setOpened?: (boolean: boolean) => void
```

Comment:
boolean is a VERY bad name for a variable (in fact it is probably should not even be allowed)

Reply:
I'm also curios as to why all of these callbacks are nullable, they sound somewhat important and I wonder if they ever are/should be null

Reply:
setOpened is also a really generic name for a function and doesn't well describe what it does.  A good rule of thumb for callbacks is to name the for the thing which happened (things like; `onUserDeleted`, `beforeTaskShutdown`, or `afterBombExploded`).

Reply:
> I'm also curios as to why all of these callbacks are nullable, they sound somewhat important and I wonder if they ever are/should be null

It should be nullable, because in Admin, we only use the closeAccordion. 
And in HackathonDetails, we only use the reload and setOpened
HackathonForm is being used in 2 different places with different argument that are being passed